src/Language/Fixpoint/Types/Config.hs

```diff
@@ -183,6 +184,7 @@ defConfig = Config {
   , nonLinCuts               = False &= help "Treat non-linear kvars as cuts"
   , noslice                  = False &= help "Disable non-concrete KVar slicing"
   , rewriteAxioms            = False &= help "allow axiom instantiation via rewriting"
+  , useInterpreter           = False &= help "Use the interpreter to assist PLE"
```

Comment:
If I'm reading this correctly, defaulting to `False` means that no test in CI rehearses this algorithm. Shouldn't it be enabled for some of the tests? The whole testsuite is fast enough that maybe it could be executed a second time with this flag enabled.